README.md

```diff
@@ -0,0 +1,62 @@
+
+# Upload your sections and articles to Zendesk  
+
+From a FileTree with folders and HTML articles, upload them to your Zendesk Help Center.  
```

Comment:
From a file tree with folders and HTML articles, upload them to your Zendesk Help Center.